.github/workflows/install-dependencies.sh

```diff
@@ -55,5 +55,11 @@ gcloud auth configure-docker --quiet
 # install pip3
 sudo apt-get install python3-pip
 
+# install unzip
+sudo apt-get install unzip
+
+# install node (for terraform)
+sudo apt-get install nodejs
+
```

Comment:
why do we need to install these if everything worked before? does it mean the terraform actions do not work otherwise?

Reply:
Previously, there was no terraform as part of the CI jobs.  Unzip is necessary to install the pre-packaged Github action, though other methods are possible.  Nodejs was necessary to run terraform on that machine.